packages/components/tooltip/src/tooltip.tsx

```diff
@@ -320,7 +320,7 @@ const Tooltip = (props: TTooltipProps) => {
             // ref accepts `LegacyRef`, which is a union of `RefObject` and `string`
             // propper.ref returns `RefObject`
             ref={popper.ref as LegacyRef<HTMLDivElement>}
-            css={{
+            style={{
```

Comment:
Getting a weird error with `css`. Since we are passing a CSS object, I think we can use `style`.

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/1110551/229819898-4cc1e3cb-4678-4a54-b28a-4a7df522a8b2.png">